application/src/main/java/org/opentripplanner/service/vehiclerental/VehicleRentalService.java

```diff
@@ -38,4 +38,14 @@ List<VehicleRentalStation> getVehicleRentalStationForEnvelope(
     double maxLon,
     double maxLat
   );
+
+  /**
+   * Gets all vehicle rental places inside an envelope.
+   */
+  List<VehicleRentalPlace> getVehicleRentalPlacesForEnvelope(
+    double minLon,
+    double minLat,
+    double maxLon,
+    double maxLat
```

Comment:
Since the method mentions the envelope can you make the it use the `Envelope` class? I know that the other method also uses the doubles but we don't want to continue this.